packages/rum-core/src/domain/rumSessionManager.spec.ts

```diff
@@ -190,172 +190,37 @@ describe('rum session manager', () => {
-        trackLongTasks: undefined,
-        expectSessionReplay: true,
-        expectResources: true,
-        expectLongTasks: true,
-      },
-      {
-        description:
-          'old WITHOUT_SESSION_REPLAY plan without trackResources/LongTasks should have no replay, no resources and no long tasks',
-        trackedWithSessionReplay: false,
-        oldPlansBehavior: true,
-        trackResources: undefined,
-        trackLongTasks: undefined,
-        expectSessionReplay: false,
-        expectResources: false,
-        expectLongTasks: false,
-      },
-      {
-        description:
-          'old WITH_SESSION_REPLAY plan with trackResources/LongTasks=false should have replay, no resources and no long tasks',
-        trackedWithSessionReplay: true,
-        oldPlansBehavior: true,
-        trackResources: false,
-        trackLongTasks: false,
-        expectSessionReplay: true,
-        expectResources: false,
-        expectLongTasks: false,
-      },
-      {
-        description:
-          'old WITHOUT_SESSION_REPLAY plan with trackResources/LongTasks=false should have no replay, no resources and no long tasks',
-        trackedWithSessionReplay: false,
-        oldPlansBehavior: true,
-        trackResources: false,
-        trackLongTasks: false,
-        expectSessionReplay: false,
-        expectResources: false,
-        expectLongTasks: false,
-      },
-      {
-        description:
-          'old WITH_SESSION_REPLAY plan with trackResources/LongTasks=true should have replay, resources and long tasks',
-        trackedWithSessionReplay: true,
-        oldPlansBehavior: true,
-        trackResources: true,
-        trackLongTasks: true,
-        expectSessionReplay: true,
-        expectResources: true,
-        expectLongTasks: true,
-      },
-      {
-        description:
-          'old WITHOUT_SESSION_REPLAY plan with trackResources/LongTasks=true should have no replay, resources and long tasks',
-        trackedWithSessionReplay: false,
-        oldPlansBehavior: true,
-        trackResources: true,
-        trackLongTasks: true,
         expectSessionReplay: false,
-        expectResources: true,
-        expectLongTasks: true,
       },
     ].forEach(
       ({
         description,
         trackedWithSessionReplay,
-        oldPlansBehavior,
-        trackResources,
-        trackLongTasks,
         expectSessionReplay,
-        expectResources,
-        expectLongTasks,
       }: {
         description: string
         trackedWithSessionReplay: boolean
-        oldPlansBehavior: boolean
-        trackResources: boolean | undefined
-        trackLongTasks: boolean | undefined
         expectSessionReplay: boolean
-        expectResources: boolean
-        expectLongTasks: boolean
       }) => {
         it(description, () => {
           configuration = {
             ...configuration,
-            trackResources,
-            trackLongTasks,
-            oldPlansBehavior,
           }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏remove this? I don't think we mutate the config in this test

Reply:
that looks like a leftover, nice catch!